experimenter/tests/integration/android/build_android.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd mobile/android/fenix \
+&& ./gradlew clean app:assembleFenixDebug \
+&& mv app/build/outputs/apk/fenix/debug/app-fenix-x86_64-debug.apk ./ \
+&& ./gradlew clean app:assembleFenixDebugAndroidTest \
+&& mv app/build/outputs/apk/androidTest/fenix/debug/app-fenix-debug-androidTest.apk ./
```

Comment:
You don't need `&&` here.

Since you have `set -e`, any command that fails will cause the script to exit.
```suggestion
cd mobile/android/fenix
./gradlew clean app:assembleFenixDebug
mv app/build/outputs/apk/fenix/debug/app-fenix-x86_64-debug.apk ./
./gradlew clean app:assembleFenixDebugAndroidTest
mv app/build/outputs/apk/androidTest/fenix/debug/app-fenix-debug-androidTest.apk ./
```